City-of-Turku/kada
# PORI-X Remove driveturku layout dependecy from features

To test:

1. drush fra -y && drush cc all
2. Go to front page and any other page on the sites navigation and make sure that you can see the layout with Pori logo. This means that all the contexts are using correct layout and Driveturku theme is no longer refered anywhere.
